EventsExpress/ClientApp/src/actions/logout.js

```diff
@@ -4,17 +4,25 @@ import { reset_hub } from './chat';
 import { resetNotification } from './chats';
 export const SET_LOGOUT = "SET_LOGOUT";
 
-  export default function logout(){
-    localStorage.clear();
-    return dispatch => { 
+export default  function logout() {
+       RevokeToken();
+       localStorage.clear();
+       return  dispatch => { 
       dispatch(reset_hub());
       dispatch(setLogout());
-      dispatch(resetNotification());
+        dispatch(resetNotification());
+       
     }
   }
 
   function setLogout() {
     return {
       type: SET_LOGOUT
     };
-  }
\ No newline at end of file
+}
+
+async function RevokeToken() {
```

Comment:
Please follow naming convention on the UI and use camel-casing.

Reply:
renamed using camel-casing